tests/acceptance/features/coreApiSharePublicLink1/accessToPublicLinkShare.feature

```diff
@@ -1,4 +1,4 @@
-@api @public_link_share-feature-required @files_sharing-app-required @issue-ocis-reva-282
+@api @public_link_share-feature-required @files_sharing-app-required @issue-ocis-reva-282 @issue-4623 @notToImplementOnOCIS
```

Comment:
```suggestion
@api @public_link_share-feature-required @files_sharing-app-required @issue-ocis-reva-282 @issue-4623 @toImplementOnOCIS
```

Reply:
But first, we need to confirm that the tests are failing due to unimplemented features in oCIS. Because I think the preview to the public link should work.

Reply:
https://github.com/owncloud/ocis/issues/5519